BaseFilter/sections/specific.txt

```diff
@@ -24,6 +24,11 @@ triblive.com###storyContent > div[data-type="float"][data-stn-player]
 !##################
 !
 ! Ad-Shield ad reinsertion
+!#if (adguard_ext_safari || adguard_app_ios || adguard_ext_android_cb)
+##a[href*=".xyz/"] img[src*="html-load.com"]
+##a[href*=".xyz/"] + img[src*="html-load.com"]
```

Comment:
As I remember, they break styles, even if ads hidden. Maybe better to make the rules domain specific?

Reply:
Also I can't reproduce at the moment.

Reply:
On my end they doesn't trigger that breaking. As commented, somehow specific didn't work but I'll test again later.

Reply:
> On my end they don't trigger that breaking

Same on another sites with Ad-Shield?

> As commented, somehow specific didn't work but I'll test again later.

Probably they must be applied in iframe with address html-load.com, and the site domain must be used (in Safari).

Reply:
Doesn't work with
```
html-load.com,jin115.com,picrew.me##a[href*=".xyz/"] img[src*="html-load.com"]
html-load.com,jin115.com,picrew.me##a[href*=".xyz/"] + img[src*="html-load.com"]
```

so not the iframe issue.

Reply:
If ever the generic caused breakage, we can exclude that site. Anyway user will see tons of ads on the site.

Reply:
> Anyway user will see tons of ads on the site

But a site will be usable. On desktop I get their error when used these rules.

Reply:
Which site? I don't know whey specific doesn't work, but maybe we can adjust the rules?

Reply:
I don't remember.

https://github.com/AdguardTeam/AdguardFilters/issues/166979 
On this site it seems they are not working on iOS.

Reply:
Hmm, can any member craft rules to hide these ads without breakage on iOS? As said, we can try the current best - few user bother to install script manager.

Reply:
I tried out the rules on my iPhone.

```
##a[href*=".xyz/"] img[src*="html-load.com"]
##a[href*=".xyz/"] + img[src*="html-load.com"]
html-load.com,jin115.com,picrew.me##a[href*=".xyz/"] > div[クラス] 
```
Screenshot 1 shows the original, and Screenshot 2 is after applying the rules. Some of the ads disappeared, but not all.

<details>

<summary>Screenshot 1 </summary>

![image](https://github.com/user-attachments/assets/17ad260e-bc4e-4c52-9f90-e2a179d1f82f)
 
</details>


<details>

<summary>Screenshot 2 </summary>

![image](https://github.com/user-attachments/assets/4dcfaf29-90e0-40b8-a910-d4fd3f31f380)
 
</details>

Reply:
I know and this is expected. I'm trying to just mitigate damage here, as totally hiding them 100% triggers anti-adblock detection.

Reply:
Ok. Let's try to add to the filter.
Please remove `adguard_ext_safari` directory - it seems scriptlet rules are working here.
And add a comment about purpose (generic - because specific rules don't help, for platforms without scriptlets)

Reply:
> https://github.com/AdguardTeam/AdguardFilters/issues/166979
> On this site it seems they are not working on iOS.

Do you see breakage or detection, or just not hiding ads?

Reply:
I mind iOS free user. Not hiding all ads is fine, but breakage is not.